mongodb.js

```diff
@@ -0,0 +1,102 @@
+const { MongoClient, Timestamp } = require("mongodb");
+
+module.exports = { db_connect, db_setup, new_print_job, new_workflow, new_workflow_step, new_simulation_report };
```

Comment:
I would rename these to: `dbConnect`, `dbSetup`, `newPrintJob`, `newWorkflow`, `newWorkflowStep`, and `newSimulationReport`.